packages/components/atoms/f-form-field/stories/Checkbox.stories.js

```diff
@@ -52,16 +39,13 @@ export const CheckboxComponent = () => ({
     },
     template:
         `<form-field
-            :locale="locale"
             :label-text="labelText"
-            input-type="checkbox"
-            :shouldShowLabelText="shouldShowLabelText"
-            :field-size="fieldSize"
+            :value="value"
             :has-error="hasError"
             :is-grouped="isGrouped"
+            :shouldShowLabelText="false"
```

Comment:
Do you need this now (as it's deleted from the options above, and I can't see it used in the component)?